src/Data/TaskerExecution.php

```diff
@@ -68,6 +68,7 @@ public function getRawData()
             'started_count' => $this->task->getStartedCount(),
             'php_version'   => str_replace(PHP_EXTRA_VERSION, '', PHP_VERSION),
             'app_version'   => $this->getAppVersionNumber(),
+            'queue_source'  => $this->task->getQueueSource(),
```

Comment:
This will be available in the next g4/tasker release. Wait until that release is created and update composer.json of this package to require that g4/tasker release (because before that release this method `getQueueSource()` does not exist.

Or better, check if the method exists on this object, so write the following:
`queue_source => method_exists($this->task, 'getQueueSource') ? $this->task->getQueueSource() : null`